Add build script to create minified build

tlvenn opened the pull request:
Hi Oleg,

A quick PR, largely inspired by GraphiQL to build the minified build.

OlegIlyenko commented on package.json:
```diff
@@ -11,10 +11,21 @@
+  },
   "scripts": {
-    "build": "babel src --out-dir dist/",
+    "build": "./build.sh",
```
maybe we can still have both definitions: `build`, `quickBuild`. The problem is that uglyfication is very slow, so for quick iteration (during development) it would be better to have a quick option.

I would suggest to move all bash scripts in `scripts` folder and have 2 build scripts as well: `build.sh`, `quickBuild.sh` WDYT?

tlvenn replied:
Done

OlegIlyenko commented on src/index.js:
```diff
@@ -1,10 +1,10 @@
+var HeaderEditor = require('./HeaderEditor')
+var KeepLastTaskQueue = require('./KeepLastTaskQueue')
+var QuerySelectionButton = require('./QuerySelectionButton')
+var config = require('./config')
```
👍

OlegIlyenko commented on build.sh:
```diff
@@ -0,0 +1,18 @@
+rm -rf dist/ && mkdir -p dist/
+babel src --out-dir dist/
+echo "Bundling graphiql-workspace.js..."
+browserify -g browserify-shim -s GraphiQL dist/index.js > graphiql-workspace.js
```
There problem here is with `-s` (standalone) option. In case of GraphiQL, it exposes `GraphiQL` react component. In our case we actually need to expose at least 2 of them: `GraphiQLWorkspace` and `AppConfig`. Not sure what would be the best way to do it: maybe export additional object in `index.js` and put these 2 component in there (let's say it;s called `graphiqlWorkspace`) then we can provide following option to browserify instead: `-s graphiqlWorkspace`

tlvenn replied:
I have renamed the export to use `graphiqlWorkspace` but I dont see any issue right now by using the standalone option. My understanding is it will export the whole export object and the name being passed is merely used for naming the global export. So you can do `graphiqlWorkspace.GraphiQLWorkspace` and `graphiqlWorkspace.AppConfig`.

Am i missing something ?

OlegIlyenko commented on build.sh:
```diff
@@ -0,0 +1,18 @@
+echo "Bundling graphiql-workspace.js..."
+browserify -g browserify-shim -s GraphiQL dist/index.js > graphiql-workspace.js
+echo "Bundling graphiql-workspace.min.js..."
+browserify -g browserify-shim -g uglifyify -s GraphiQL dist/index.js 2> /dev/null | uglifyjs -c --screw-ie8 > graphiql-workspace.min.js 2> /dev/null
```
the same here

tlvenn replied:
Fixed

tlvenn commented:
Hi @OlegIlyenko, sorry for the late follow up on this PR. I believe i have addressed your comments. I am still unclear on the whole browserify standalone issue you have raised however as commented above.

tlvenn commented:
Just realized my code comment is hidden now i have committed some updates..
You can see it here: https://github.com/OlegIlyenko/graphiql-workspace/pull/2#discussion_r97223226

OlegIlyenko commented:
Great, thanks a lot!
